packages/bun-extensions/src/load-php.ts

```diff
@@ -0,0 +1,18 @@
+import { plugin } from 'bun';
+
+plugin({
+	name: 'PHPTextPlugin',
+	setup(build) {
+		build.onLoad({ filter: /\?(.*&|)raw(&.*|)$/ }, async (args) => {
```

Comment:
even for a humble draft like this a tiny example comment above the regex pattern can clear headache. what is it testing against (e.g. a URL?) and what is it looking for (query args?)?

Reply:
Doc added:

```ts
/**
 * Adds support for `import contents from "file.php?raw"`.
 *
 * It matches Vite's behavior, which is to read the file as text
 * and return its textual content. This allows us to run Bun and
 * Vite on the same codebase.
 */
```